.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build main
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        java: [11, 8]
```

Comment:
Why are you building both Java 8 and Java 11? There is no reason to build Java 11.

Reply:
You could say the same about java 8.
Why build with it? Sooner or later will PaperMC abandon it

Reply:
Yes, but this is not a Paper-only plugin. There's no reason to build with both, as Java 8 builds will work on Java 11+. If Spigot *ever* switches to J11+ (they won't) then the version we build against can be bumped. There is no reason to build both.

Reply:
Then just keep it at 11 and make targetCompatibility 8.

11 is backwards compatible with 8, so changing target and/or source is fine.